### ISSUE TITLE
[bitnami/matomo] archive cronjob isn't created

### ISSUE THREAD

**Issue by JaZo** [bitnami/charts#21266]
### Name and Version

bitnami/matomo 3.2.0

### What architecture are you using?

amd64

### What steps will reproduce the bug?

Setup an environment with default values.

### Are you using any custom parameters or values?

No

### What is the expected behavior?

I'd expect the archive cronjob to be created.

### What do you see instead?

There is no archive cronjob.

### Additional information

Looks like it's caused by #21232

**Comment by viceice** [comment]
I think this will be fixed by #21268. I'm seeing errors like this when using `extraVolumes`

```
Helm upgrade failed: error while running post render on files: map[string]interface {}(nil): yaml: unmarshal errors: line 90: mapping key "apiVersion" already defined at line 2 line 91: mapping key "kind" already defined at line 3 line 92: mapping key "metadata" already defined at line 4 line 101: mapping key "spec" already defined at line 15 Last Helm logs: preparing upgrade for matomo resetting values to the chart's original version
```

**Comment by carrodher** [comment]
Thank you for opening this issue and submitting the associated Pull Request. Our team will review and provide feedback. Once the PR is merged, the issue will automatically close.

Your contribution is greatly appreciated!
